flybywiresim/a32nx
# refactor: remove unused a32nx_core module, improve some LocalVarUpdater functions

## Summary of Changes

* Removes unused Core module `A32NX_Ground_Reference`, which is also unnecessary, since the LVar it updates is essentially the same as the simvar `GROUND ALTITUDE`
* Removes unnecessary Simvar call from LocalVarUpdater `_flapsInMotionSelector` function
* Removes redundant update-throttling code in `_isPacksOneSupplying`, this is now built into the LocalVarUpdater module since #4077


## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

* Make sure that the flap sounds work as previously
* Make sure that the packs sounds work as previously


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
